packages/block-editor/src/components/tabbed-sidebar/style.scss

```diff
@@ -6,7 +6,7 @@
 	overflow: hidden;
 
 	@include break-medium() {
-		width: 350px;
```

Comment:
I think a simpler plan will be to:
- When pattern categories are open
- Update this width to be the desired total width
- Remove all the absolute hacks for the pattern categories